apps/public-docsite-v9/src/Utilities/Theme/createCSSRuleFromTheme/index.stories.tsx

```diff
@@ -6,7 +6,7 @@ import bestPracticesMd from './createCSSRuleFromThemeBestPractices.md';
 export { Default } from './createCSSRuleFromThemeDefault.stories';
 export { Switching } from './createCSSRuleFromThemeSwitching.stories';
 
-export default {
+const meta: Meta = {
```

Comment:
A random change to trigger the CI pipeline since the `.storybook/docs-root.css` is an excluded path in the pipelines.

TODO: Remove this before merging.

Reply:
FYI: this trigger shouldn't be needed.

`build-storybook` invalidas cache if anything in root `.storybook/**` is changes, thus public docsite should have been build without this. if that's not the case we need to fill regression bug to nx.

 https://github.com/microsoft/fluentui/blob/843386fc13e12d9efca29eb1e26a2654239167b6/nx.json#L28-L29

Reply:
after removing artificial commit - we get pipeline runs as expected

<img width="1031" alt="image" src="https://github.com/user-attachments/assets/242e3677-e8c2-4423-a331-b26770ee3d63">

Reply:
I think the issue was not related to nx, the azure pipelines were skipped for some reason.

Reply:
<img width="1471" alt="Screenshot 2024-08-21 at 15 35 28" src="https://github.com/user-attachments/assets/cf19db8d-c15d-418b-ad53-edb9127de963">